### ISSUE TITLE
Skeleton component

### ISSUE THREAD

**Issue by joneff** [telerik/kendo-themes#2288]
Design: https://github.com/telerik/kendo-ux-private/issues/274

**Comment by CheetahDev** [comment]
Hi @joneff,

I'm curious about this upcoming feature.

As we didn't had this component, we have added one from Ionic (ion-skeleton).

For the time being, we have custom components (date, text, textarea, ...) that wrap fields provided by Kendo UI.

This allow us to manage a loading state and thus to display the ion-skeleton on top of Kendo UI fields.

It would be really nice if the new skeleton could be "built-in" Kendo UI fields (either inputs or text value / labels).

Available to share more details if needed.

Regards

**Comment by GeorgeGindev** [comment]
## Skeleton Overview Demo
Test sample - [https://dojo.telerik.com/IhiZERED](https://dojo.telerik.com/IhiZERED)

## Proposed Rendering
```html
  <div class="k-skeleton-grid [k-animation-pulse|wave]">
    <div class="k-skeleton-text" style="grid-column: [start] / [end]; grid-row: [start] / [end];"></div>
    <div class="k-skeleton-circle" style="grid-column: [start] / [end]; grid-row: [start] / [end];"></div>
    <div class="k-skeleton-rect" style="grid-column: [start] / [end]; grid-row: [start] / [end];"></div>
  </div>
```

**Comment by joneff** [comment]
Closed by #2405

**Comment by joneff** [comment]
@CheetahDev for the time being, we are providing the classes only and components in the different suites (jquery, angular, react, vue, blazor).

While I can't comment on the exact implementation yet, it will be possible to have something similar to `ionic-skeleton`. We are looking to further improve the styles and components, so we could as well provide integration with other components.

By integration I mean, and don't take the code literally, something like `<Button isLoading={{true}} />`. Again, that's yet to be seen.
